apps/chat/agent/tools.py

```diff
@@ -171,6 +181,37 @@ def action(self, key: str, value: Any):
         return "Success"
 
 
+class AttachMediaTool(CustomBaseTool):
+    name: str = AgentTools.ATTACH_MEDIA
+    description: str = "Attach a media file to your response"
+    requires_session: bool = True
+    args_schema: type[schemas.AttachMediaSchema] = schemas.AttachMediaSchema
+    callback: Callable | None = None
```

Comment:
shouldn't this be a required field? i.e. not None?

Also you can add type hints to callable:
```python
Callable[[int], Any]
```